lib/ma_crossover/events/life_start.js

```diff
@@ -1,6 +1,12 @@
 'use strict'
 
+const fs = require('fs')
+const os = require('os')
 const HFI = require('bfx-hf-indicators')
+const checkHeader = require('../util/check_header')
+const dir = `${os.homedir()}/.honeyframework/ma_crossover.csv`
```

Comment:
so far the repo was a library that did not knew that the app has a settings directory... hmmm...

how about passing a logdir option to algo_host? also did you think of the hosted version?